src/main.rs

```diff
@@ -168,16 +163,16 @@ async fn main() {
             peer_addr: addr,
         };
 
-        let cloned_ctx = ctx.clone();
+        //let cloned_ctx = ctx.clone();
```

Comment:
Please don't leave this commented code lying around. If it's no longer needed, remove it.